index.bs

```diff
@@ -2246,6 +2249,13 @@ the [RP] know that the extension was honored by the client. Similarly, any exten
 specify a method of augmenting the [=authenticator data=] to let the [RP] know that the extension was honored by the
 authenticator.
 
+If an extension is designed to enable the [RP] to know whether it was understood and acted on,
+the extension MUST return some data from the client to the [RP] for this purpose.
+If an extension does not otherwise require any result values, it SHOULD be defined as returning a Boolean extension result,
+set to `true` to signify that the extension was understood and processed.
+Likewise, such an extension that is processed by the authenticator that does not otherwise require any result values
+SHOULD also return a Boolean extension result,
+set to `true` to signify that the extension was understood and processed.
```

Comment:
per prior discussion with MikeJ @selfissued, I am inclined to model the above-suggested "the extension was honored" return value as a single bit (or byte) signifying "acknowledged", i.e., a value signifying "ACK" and not as a boolean, because we never intend to return false, and that seems potentially confusing to me.